assign2/src/components/Secondassign.jsx

```diff
@@ -0,0 +1,80 @@
+import React,{ useState } from 'react'
+
+//when we do not declair setEmail and pass func seperately it will get reset when on change ends
+
+
+const Secondassign = () => {
+
+
+  function showIdPass(){
+    let promptName = prompt(`if the password does not match,it wont show on console`, `${firstName} ${lastName}`)
+
+    if(password==passwordAgain){
+    return(
+       <div>
+        {console.log(`First Name: ${firstName}`)}
+           {console.log(`Last Name: ${lastName}`)}
+           {console.log(`email: ${email}`)}
+           {console.log(`password: ${password}`)}
+          
+       </div>)}
+     
+   }
+
+   function setEmailFunc(e){
```

Comment:
You can try creating objects and have a single setState method

Reply:
okay.